ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py

```diff
@@ -614,6 +614,38 @@ def test_tensorflow_Atan(  # NOQA
     )
 
 
+# AvgPool
+@handle_frontend_test(
+    fn_tree="tensorflow.raw_ops.AvgPool",
+    data_format=st.sampled_from(["NHWC", "NCHW"]),
+    x_k_s_p=helpers.arrays_for_pooling(min_dims=4, max_dims=4, min_side=1, max_side=4),
+    test_with_out=st.just(False),
+)
+def test_tensorflow_AvgPool(  # NOQA
+    *,
+    x_k_s_p,
+    data_format,
+    frontend,
+    test_flags,
+    fn_tree,
+    on_device,
+):
+    input_dtype, x, kernel, strides, pad = x_k_s_p
+    data_format = data_format
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        frontend=frontend,
+        test_flags=test_flags,
+        fn_tree=fn_tree,
+        on_device=on_device,
+        value=x,
```

Comment:
This should be `x[0]`, since `x` is a list containing an array and we only want to pass the array itself.